src/nodes/core/IndexNode.js

```diff
@@ -62,11 +66,13 @@ class IndexNode extends Node {
 IndexNode.VERTEX = 'vertex';
 IndexNode.INSTANCE = 'instance';
 IndexNode.DRAW = 'draw';
+IndexNode.LOCAL = 'local';
 
 export default IndexNode;
 
 export const vertexIndex = nodeImmutable( IndexNode, IndexNode.VERTEX );
 export const instanceIndex = nodeImmutable( IndexNode, IndexNode.INSTANCE );
+export const localInstanceIndex = nodeImmutable( IndexNode, IndexNode.LOCAL );
```

Comment:
Can you rename to `instanceLocalIndex`? It looks closer to the TSL names like `normalLocal`, `positionLocal`, ....

Reply:
I can change it, but I was wondering if it might be better as something like invocationLocalIndex, to align it with WGSL syntax and make it more clear it's only for compute purposes.

Reply:
Yes, `invocationLocalIndex` seems better for this context.